source/documentation/using_the_api/item_entry_record_example.md

```diff
@@ -0,0 +1,24 @@
+## The entry/item/record relationship
+
+An **entry** expresses a change in a register. For example, using the `GET /entries` endpoint for the `country` register will return all entries there have ever been to that register. The very [first entry to the `country` register](https://country.register.gov.uk/entries/1.json) introduced a change for the **key** ‘SU’, corresponding to the country of the USSR. 
```

Comment:
In the frontend we use the phrase `update` instead of `change`

Reply:
Good spot @danmitchell- and thanks for the review. I'll change this now.

Just FYI I'm waiting on approval from the tech writing team before I can publish this.